lib/cinder

```diff
@@ -442,6 +442,11 @@ function init_cinder {
 function install_cinder {
     git_clone $CINDER_REPO $CINDER_DIR $CINDER_BRANCH
     setup_develop $CINDER_DIR
+    if is_clearlinux; then
+	install_package openstack-common
```

Comment:
This should be done by adding openstackcommon to files/rpms-clear/general.  None of the "install_package openstack-common" is required then.

Reply:
Will be fixed in the next patchset.